db_setup/upload_seed_data.py

```diff
@@ -0,0 +1,112 @@
+"""This script seeds the database plant, location and duty tables with .csv files."""
+
+
+import csv
+from os import environ
+
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, sql
+from sqlalchemy.engine.base import Connection
+
+
+load_dotenv()
+
+
+def get_database_engine():
+    """Returns the database engine."""
+    try:
+        engine = create_engine(
+            f"mssql+pymssql://{environ['DB_USER']}:{environ['DB_PASSWORD']}@{environ['DB_HOST']}:{environ['DB_PORT']}/{environ['DB_NAME']}?charset=utf8"
+        )
+        return engine
+    except Exception as e:
+        print(f"Error creating database engine: {e}")
+        raise e
+
+
+def upload_locations(conn: Connection, locations: list) -> None:
+    """Seeds database with location data."""
+    try:
+        conn.execute(sql.text(f"USE {environ['DB_NAME']};"))
+        conn.execute(sql.text("BEGIN TRANSACTION;"))
+
+        for row in locations:
```

Comment:
Eventually we can insert like we have in the load file, as it reduces a lot of lines, but looks good for now